src/components/textToSpeech/js/TextToSpeech.js

```diff
@@ -260,6 +266,12 @@ fluid.textToSpeech.requestControl = function (that, control, change) {
     }
 };
 
+fluid.textToSpeech.getVoiceByLang = function (that, lang) {
+    lang = lang || "en-US";
```

Comment:
rather than hardcoding to "en-US" this should use [Navigator.languages](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/languages) or [Navigator.language](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/language) with a final fallback to something that is configured in the options.